framework_tvb/tvb/core/services/burst_service.py

```diff
@@ -27,21 +27,25 @@
 #   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
 #
 #
-
+import json
 import os
 from datetime import datetime
+
+from tvb.adapters.datatypes.db.mapped_value import DatatypeMeasureIndex
+from tvb.adapters.datatypes.h5.mapped_value_h5 import DatatypeMeasureH5
```

Comment:
These are a bad dependency. we should not import tvb.adapters from tvb.core.services.
Maybe move these datatypes in the core part, as they are not adapter specific, but basic ones for our workflow?

Reply:
How about moving only DatatypeMeasureH5 under core/entities/file/simulator and determining the proper index from REGISTRY?

Reply:
I think moving just the `DatatypeMeasureH5` is a good idea. 
But it should then be split in a separate python module file. `ValueWrapper` is a typical adapter datatype, it is not related to the simulator (as it is the `DatatypeMeasure` resulting from a PSE)

Reply:
Good point, they are split now